docs/resources/Channel.md

```diff
@@ -332,6 +332,34 @@ Discord utilizes a subset of markdown for rendering message content on its clien
 
 Using the markdown for either users, roles or channels will mention the target(s) accordingly.
 
+## URL Formatting
```

Comment:
Actually, I think the entire `## Message Formatting` section should be moved to the `Reference.md` page along with this section. It's general information and not specific to channel.